src/sentry/tasks/post_process.py

```diff
@@ -145,6 +165,21 @@ def handle_owner_assignment(job):
             # - we tried to calculate and could not find issue owners with TTL 1 day
             # - an Assignee has been set with TTL of infinite
             with metrics.timer("post_process.handle_owner_assignment"):
+
+                with sentry_sdk.start_span(op="post_process.handle_owner_assignment.ratelimited"):
+
+                    ratelimit_key = f"issue_owner_assignment_ratelimit:{project.id}"
```

Comment:
If you do not reuse the key in other parts of the code I would place it within `ratelimit_key` and just pass `project.id`.